### PR TITLE
Update hyscale-spec-reference.md

### PR DESCRIPTION
**Pull request by Nishanthnishu**

### CODE REVIEW

File: docs/hyscale-spec-reference.md [hunk 1]
```diff
@@ -495,6 +495,20 @@ List of k8s snippets that needs to be patched on the generated manifest files.
...
+  <tr>
+   <td><a href="#loadBalancer">loadBalancer</a>
+   </td>
+   <td>list
+   </td>
+   <td>
+   </td>
+   <td><em>Optional</em> 
+<p>   
+    <em>Can be overridden</em>
```
Comment (tushar-tushar) [inline docs/hyscale-spec-reference.md:507]: Since this can be overridden via profile, add this field in profile documentation also

**Review by tushar-tushar — COMMENTED** [review]
Update the version to 0.6.5.
Update the changelog file as well

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# v0.6.5 Release
+
+### Features:
+
+* Added loadBalancer field in spec to Load Balancer Support
```
Comment (tushar-tushar) [inline CHANGELOG.md:5]: Added loadbalancer field in spec to <provide> Load balancer Support.
OR Added loadbalancer field in spec <for supporting> Load balancer configuration

Comment (Nishanthnishu) [reply CHANGELOG.md]: Added loadBalancer field in spec for Load Balancer Support

**Review by tushar-tushar — APPROVED** [review]
LGTM

**Pull request merged**